tests-web/assets/js/tests.js

```diff
@@ -0,0 +1,24 @@
+import { makeOsmLayer, makeJsonLayer, makeDotLayer  } from './layers.js';
+
+/** Load and display all the files associated with a test case. */
+export const makeOpenTest = (map) => async (name) => {
+    const prefix = `/src/${name}/`;
```

Comment:
Answering one of my questions from earlier... looks like the directory should be dragged over